src/index.tsx

```diff
@@ -1,14 +1,31 @@
 import { createRoot } from 'react-dom/client';
-import { HashRouter as Router } from 'react-router-dom';
+import {
+  Navigate, Route, HashRouter as Router, Routes,
+} from 'react-router-dom';
 
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 
 import { App } from './App';
+import { HomePage } from './components/HomePage/HomePage';
+import { PeoplePage } from './components/PeoplePage/PeoplePage';
+import { NotFoundPage } from './components/NotFoundPage/NotFoundPage';
 
 createRoot(document.getElementById('root') as HTMLDivElement)
   .render(
     <Router>
-      <App />
+      <Routes>
```

Comment:
nb: as for me, better to add all routes directly in the `<App />` component, because in the root (index.ts) file, better to add some wrappers like Routes, Redux provider, context etc